pkg/networkmanager/interfaces.js

```diff
@@ -2596,6 +2612,19 @@ PageNetworkInterface.prototype = {
                                                        dev.DeviceType == 'bridge'));
         $('#network-interface-delete').toggle(is_deletable && managed);
 
+        function render_interface_section_separator(title) {
```

Comment:
@andreasn and I didn't immediately like these new titles.  In any case, they should be visually less 'busy', without border or background.  Maybe two columns?